superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -539,9 +539,13 @@ class SqlEditor extends React.PureComponent {
   }
 
   renderQueryLimit() {
-    const menuDropdown = (
+    // Adding SQL_MAX_ROW value to dropdown
+    const { maxRow } = this.props;
```

Comment:
can we add some tests and check if `maxRow` is undefined? Will it print "undefined" as an option?